pkg/virtctl/imageupload/imageupload.go

```diff
@@ -926,3 +946,56 @@ func handleEventErrors(client kubecli.KubevirtClient, pvcName, dvName, namespace
 
 	return nil
 }
+
+func createNewDataSource(client kubecli.KubevirtClient, name, namespace string) error {
+	ds := &v1beta1.DataSource{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+		},
+		Spec: v1beta1.DataSourceSpec{
+			Source: v1beta1.DataSourceSource{
+				PVC: &v1beta1.DataVolumeSourcePVC{
+					Name:      name,
+					Namespace: namespace,
+				},
+			},
+		},
+	}
+	setDefaultInstancetypeLabels(ds.Labels)
+
+	_, err := client.CdiClient().CdiV1beta1().DataSources(namespace).Create(context.Background(), ds, metav1.CreateOptions{})
+	if err == nil {
```

Comment:
nit: Inline this

Reply:
Yeah I don't like long lines, it's hard to read it on my screen.

I'll update it.

Reply:
It won't work here, because `err` needs to be returned by the end:
```
...
	_, err := client.CdiClient().CdiV1beta1().DataSources(namespace).Create(context.Background(), ds, metav1.CreateOptions{})
	if err == nil {
		fmt.Printf("Created a new DataSource %s/%s\n", namespace, name)
	}
	return err
}
```